### PR TITLE
s47029172 - ViT Classification of ADNI dataset

### PR DESCRIPTION
**Pull request by Talia-12**
The `ViT-Classifier-s4702917` folder contains my implementation of a ViT classifier for the ADNI dataset for Alzheimer’s disease. I have taken a pre-trained model provided by the `timm` library and done transfer learning to optimise it for the ADNI dataset. The final accuracy I was able to achieve in classifying the test set was only 62.3% after attempting a variety of different hyperparameters and variations on the model, and I believe that this is probably around the best that a pure transformer architecture can do with this dataset.

### CODE REVIEW

**Comment by LinfengLiu98** [comment]
**This is an initial inspection, no action is required at this point**

Difficulty: Hard

Readme:
Excellent

Commit messages: Excellent, detailed.

Code:
- Uses vision transformer
- Well commented
- Good model design

Functionality/Performance:
- Test ACC: 62.3% 
- Preprocssing data correctly
- Did patient level split.
- Well done

**Comment by shakes76** [comment]
# Marking
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate design and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem 
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage
Module present
Commenting
No Data leakage
Difficulty: Hard
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
ReadMe acceptable missing both usage and refs -2
Model/technical explanation
Good Description and Comments 
Markdown used and PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good
